pallets/usage/src/tests.rs

```diff
@@ -205,6 +226,72 @@ fn ocw_interval_maintained() {
     })
 }
 
+#[test]
+fn ocw_double_charge() {
+    init_logger();
+    let (mut ext, pool) = with_offchain_ext();
+    ext.execute_with(|| {
+        // Reserve some currency on users' accounts
+        for i in 1u64..=10 {
```

Comment:
No need to set the type explicitly as it can be inferred.
```suggestion
        for i in 1..=10 {
```

Reply:
ok